langserver/langserver.py

```diff
@@ -99,10 +101,11 @@ def workspace_modules(self, path) -> List[Module]:
     def workspace_symbols(self):
         if self.symbol_cache:
             return self.symbol_cache
-        symbols = []
         py_paths = (path for path in self.fs.walk(self.root_path) if path.endswith(".py"))
-        for path, src in self.fs.batch_open(py_paths):
-            symbols.extend(extract_symbols(src, path))
+        py_srces = self.fs.batch_open(py_paths)
+        with multiprocessing.Pool() as p:
+            symbols_chunks = p.imap_unordered(extract_symbols_star, py_srces)
```

Comment:
I would assume this is more performant since:
* It doesn't require loading all source into RAM
* It has no ordering requirements

Playing with chunk_size may give us more perf. I'll experiment with that now on my laptop before shipping.